app/src/main/res/values/strings.xml

```diff
@@ -927,4 +927,9 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_pedestrian_crossing_island">"Does this pedestrian crossing have an island?"</string>
     <string name="quest_surface_detailed_answer_impossible_title">Describe surface</string>
     <string name="open_url">Open URL</string>
+    <string name="quest_tactile_paving_kerb_title">Is there tactile paving on the kerb on this crossing?</string>
+    <string name="quest_kerb_height_title">What is the kerb height on this crossing?</string>
```

Comment:
Actually, shouldn't it say "What is the type/height of this **kerb**", rather than crossing? After all, we ask for each kerb individually.